main.py

```diff
@@ -351,72 +352,60 @@ async def on_ready() -> None:
     print("------")
 
 
-@client.event
-async def on_message(message: Any) -> None:
+@client.command()
```

Comment:
Just a small note: This will probably eliminate the aliases `!ross` and `!br`. I think you can pass aliases into the command decorator like this:
```python
@client.command(aliases=['ross', 'br'])
```